extension/js/common/ui/backup-ui/backup-ui-manual-module.ts

```diff
@@ -63,6 +65,28 @@ export class BackupUiManualActionModule extends BackupUiModule<BackupUi> {
     }
   };
 
+  public doBackupOnDesignatedMailbox = async (
+    acctEmail: string,
```

Comment:
probably `acctEmail` param isn't needed and we can use `this.ui.acctEmail` instead?

Reply:
oh - yes, can remove the passed parameter and use `this.ui.acctEmail`.